engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/preparation/JobPreparationActor.scala

```diff
@@ -10,85 +10,111 @@ import cromwell.core.callcaching.docker._
 import cromwell.core.logging.WorkflowLogging
 import cromwell.engine.workflow.lifecycle.execution.WorkflowExecutionActorData
 import cromwell.engine.workflow.lifecycle.execution.preparation.CallPreparation._
-import cromwell.engine.workflow.lifecycle.execution.preparation.JobPreparationActor._
+import cromwell.engine.workflow.lifecycle.execution.preparation.JobPreparationActor.{DockerNoResponseTimeout, _}
+import cromwell.services.keyvalue.KeyValueServiceActor.{KvGet, KvJobKey, KvResponse, ScopedKey}
 import wdl4s._
 import wdl4s.values.{WdlString, WdlValue}
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
-case class JobPreparationActor(executionData: WorkflowExecutionActorData,
-                                     jobKey: BackendJobDescriptorKey,
-                                     factory: BackendLifecycleActorFactory,
-                                     dockerHashingActor: ActorRef,
-                                     initializationData: Option[BackendInitializationData],
-                                     serviceRegistryActor: ActorRef,
-                                     ioActor: ActorRef,
-                                     backendSingletonActor: Option[ActorRef])
-  extends FSM[JobPreparationActorState, Option[JobPreparationActorData]] with WorkflowLogging with DockerClientHelper {
-
-  lazy val workflowIdForLogging = workflowDescriptor.id
-  
+class JobPreparationActor(executionData: WorkflowExecutionActorData,
+                          jobKey: BackendJobDescriptorKey,
+                          factory: BackendLifecycleActorFactory,
+                          val dockerHashingActor: ActorRef,
```

Comment:
Yeah, I was asking specifically about the val.  I didn't notice the dropping of the case, gotcha.